internal/internal.go

```diff
@@ -40,7 +42,7 @@ var (
 )
 
 // HealthChecker defines the signature of the client-side LB channel health checking function.
-type HealthChecker func(ctx context.Context, newStream func() (interface{}, error), reportHealth func(bool), serviceName string) error
+type HealthChecker func(ctx context.Context, newStream func(string) (interface{}, error), reportHealth func(connectivity.State), serviceName string) error
```

Comment:
Do you think it makes sense to document this function more in terms of explaining what each of the arguments do. The first time I read this code, I had to read the full implementation of the health checking code to understand what each of the arguments (specifically the callback args) meant.

Reply:
Done